lib/use-cases/check-auth.js

```diff
@@ -0,0 +1,12 @@
+class CheckAuth {
+  constructor({allowedGroups, jwt}) {
+    this.allowedGroups = allowedGroups;
+    this.jwt = jwt;
+  }
+
+  execute({token}) {
+    const payload = this.jwt.decode(token)
```

Comment:
I'm guessing we verify the token elsewhere, In the customer authorizer?

Reply:
The authorizer takes care of that